### PR TITLE
Portfolio-details-popup-window-pull-request

### PR DESCRIPTION
**Pull request by UsamaMasood12**
In this project, I added the popup window of the detail section of the project with the help of Js code.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -17,3 +17,133 @@ MOBILE_MENU_LINKS.forEach((element) => {
...
+    SEE_PROJECT_BUTTON.classList.add('see-project-card-template');
+    SEE_PROJECT_BUTTON.textContent = 'See Project';
+    SEE_PROJECT_BUTTON.addEventListener('click', function (){
+        ViewHideProject(project_info);
+    });
+    SEE_PROJECT_DIV.appendChild(SEE_PROJECT_BUTTON);
+}
+for(let i = 0; i < PROJECTS_INFO.length; i++){
+    CreateProjectCard(PROJECTS_INFO[i]);
+}
```
Comment (emmiiorji) [inline index.js]: As a best practice in JavaScript, a forEach loop is preferred over the conventional for loop especially when the sample size is not considered large. This is for better readability.

With a forEach loop, the above code could be `PROJECTS_INFO.forEach((project) => {createProjectCard(project)})`

File: index.js [hunk 2]
```diff
@@ -17,3 +17,133 @@ MOBILE_MENU_LINKS.forEach((element) => {
...
+        img_src: "./rsc/images/Img-Placeholder.png"
+    },
+    {
+        header: "Profesional Art Printing Data",
+        paragraph: "A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard",
+        technologies: ['html', 'css', 'javascript'],
+        img_src: "./rsc/images/Snapshoot-Portfolio-Desktop.png"
+    }
+];
+function CreateProjectCard(project_info) {
```
Comment (emmiiorji) [inline index.js]: As is the convention in JavaScript, kindly modify the name of the function to have a camelCase.

File: index.js [hunk 3]
```diff
@@ -17,3 +17,133 @@ MOBILE_MENU_LINKS.forEach((element) => {
...
+                <li class="card-technology first-technology">html</li>
+                <li class="card-technology">bootstrap</li>
+                <li class="card-technology">Ruby</li>
+            </ul>
+            </div>
+            <div class="see-project-container-card-template">
+            <button class="see-project-card-template">See Project</button>
+            </div>
+        </div>
+    */
```
Comment (emmiiorji) [inline index.js]: To make the code base concise, kindly remove unused pieces of code.

**Review by emmiiorji — CHANGES_REQUESTED** [review]
Hi @UsamaMasood12, 
# Status: Changes Required ♻️

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

A little bit more

## Good Highlights✔️

- Linter checks are passing
- Good commit messages
- Adhered to GitHub flow


## Required Changes ♻️
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Owusu-Desmond — APPROVED** [review]
Hi @UsamaMasood12 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Great working on the setup of the project ✅
📌 Good commit messages ✅
📌 Professional README file ✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**